assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -810,6 +827,38 @@ function initializeDiscourseCalendar(api) {
     }
   }
 
+  function _setupTimezoneOffsetButton(resetDynamicEvents) {
+    const timezoneWrapper = document.querySelector(
+      ".discourse-calendar-timezone-wrap"
+    );
+    const timezoneButton = document.createElement("button");
+
+    timezoneButton.classList.add("fc-timezoneOffset-button");
+    timezoneButton.classList.add("fc-button");
```

Comment:
```suggestion
    timezoneButton.classList.add("fc-timezoneOffset-button", "btn", "btn-icon");
```

I'd recommend using the button classes from core ("btn" and "btn-default") here. It makes the button styling more consistent, and you shouldn't need to add an active state class (below) or specific SCSS styling, it will inherit from core. 

Not sure this works 100%. Make sure you also look at the styleguide, `http://localhost:4200/styleguide/atoms/buttons`, it has great examples of button types and button states.

Reply:
Also, for accessibility, we should also add a `title` or an `ariaLabel` property to the button.